### ISSUE TITLE
Transparent dialogs on thunar and nemo

### ISSUE THREAD

**Issue by xzsk2** [vinceliuice/Colloid-gtk-theme#109]
I found this issue in [catppuccin/gtk](https://github.com/catppuccin/gtk), which is based on Colloid. I installed version 0546af8 of Colloid and figured out that the issue may originate from here. For more details, check out catppuccin/gtk/issues/89.

![图片](https://user-images.githubusercontent.com/20642596/231704358-d70e3260-f50b-4246-86fb-81b2c11412d6.png)

**Comment by vinceliuice** [comment]
I think its already fixed

**Comment by Aleksanaa** [comment]
No, it's not. Tested with e57f422bd340ecd48aea96cf1246ae79e0621852.

**Comment by Aleksanaa** [comment]
This bug is introduced between f4dcfdac62ded4bf1b0623503266e35020480e5e and 2050abd4d6ba916048c9bbdb3114eb337462988b.

**Comment by vinceliuice** [comment]
![截图 2023-06-12 01-01-40](https://github.com/vinceliuice/Colloid-gtk-theme/assets/7604295/f2129d2f-a167-439f-837b-77c91f04a8e6)
It's fixed already

**Comment by vinceliuice** [comment]
I also test catpuccin theme, it's also fixed
![截图 2023-06-12 01-11-04](https://github.com/vinceliuice/Colloid-gtk-theme/assets/7604295/370f3dec-3e61-4e93-935f-3f973a567846)

**Comment by vinceliuice** [comment]
https://github.com/catppuccin/gtk/releases

**Comment by x140x1n** [comment]
I'm also having this issue with dialogs, tried with the latest release and cloning the current main branch. Both with the same result.

**Comment by sneexy-boi** [comment]
This seems to be related to wayland compositors by the way (as someone using hyprland and I have this issue)
Dialogs aren't transparent in GNOME or KDE for example but they are in Hyprland

**Comment by diniamo** [comment]
I still have this on Hyprland, while other GTK themes work fine.

**Comment by sakarie9** [comment]
#135 Seems this PR fixes the issue

![图片](https://github.com/vinceliuice/Colloid-gtk-theme/assets/20642596/2717e87e-5bd3-48e0-8871-8e0a332e07da)
